static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -327,6 +333,7 @@ function BaseDraggableTabList({
         state={state}
         className={className}
         onReorder={onReorder}
+        onReorderComplete={() => onReorderComplete?.()}
```

Comment:
do we need the optional chaining here? looks like we always expect `onReorderComplete` to be passed in?